src/sortStudents.ts

```diff
@@ -1,16 +1,118 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'average grade',
+}
+
+export type SortOrder = 'asc' | 'desc';
+
+function getAverageGrade(arrayOfGrades: number[]): number {
+  return arrayOfGrades.reduce((prevResult, currentGrade) => {
+    return (prevResult + currentGrade);
+  }, 0) / arrayOfGrades.length;
+}
+
+function sortByName(objectOfStudents: Student[], order: string): Student[] {
```

Comment:
why `objectOfStudents`? i guess it is an array, so maybe `arrayOfStudents` will be a better name. But i suggest just using `students`. It is simple and descriptive enough.
```suggestion
function sortByName(students: Student[], order: string): Student[] {
```

Reply:
Agree, `students` looks better